moc_openstack_api_reporting.py

```diff
@@ -0,0 +1,586 @@
+#!/usr/bin/python3
+""" This is intended to poll openstack for changes to be reported in xdmod """
+# pylint: disable=line-too-long
+
+import json
+import argparse
+import logging
+import datetime
+import re
+import openstack
+import copy
+import os
+from novaclient import client as nova_client
+from cinderclient import client as cinder_client
+from keystoneclient import client as keystone_client
+import pprint
```

Comment:
There are a bunch of unused imports here:

```
moc_openstack_api_reporting.py:9:1: F401 're' imported but unused
moc_openstack_api_reporting.py:12:1: F401 'os' imported but unused
moc_openstack_api_reporting.py:15:1: F401 'keystoneclient.client as keystone_client' imported but unused
moc_openstack_api_reporting.py:16:1: F401 'pprint' imported but unused
```

Not critical, but worth fixing if you're making other changes.

Reply:
I've removed them despite the utility that I occasionally find in just keeping them around when I am developing things.